.drone.yml

```diff
@@ -0,0 +1,95 @@
+kind: pipeline
```

Comment:
Can you remove this file? Looks like it was accidentally commited (feel free to add to .gitignore)

Reply:
Yes, needed that for our CI. will clean up.